src/utils/appDataClient.ts

```diff
@@ -52,6 +52,15 @@ export const getHotspots = async () => {
   return newHotspotList.takeJSON(1000)
 }
 
+export const getValidators = async () => {
+  Logger.breadcrumb('getValidators', breadcrumbOpts)
+  const address = await getAddress()
+  if (!address) return []
+
+  const newValidatorsList = await client.account(address).validators.list()
+  return newValidatorsList.takeJSON(1000)
```

Comment:
Does this api call have a cursor? If not maybe we should use the `MAX` variable instead to make it more clear?

Reply:
I changed it to max and a few others that should be using too.